### ISSUE TITLE
Hi, I found that hinge loss doesn't work well, is that right?

### ISSUE THREAD

**Issue by CoreloneH** [asarigun/TransGAN#2]

**Comment by asarigun** [comment]
Hi @CoreloneH, thanks for your interest in the repo. Definitely! I tried to tune hyperparameters in hinge loss but the FID score didn't converge and qualitative results were terrible. So, I tried ```wgangp_eps``` and in a few epochs, it converged very well compared to hinge. You can find the details in ```train.py```.

**Comment by CoreloneH** [comment]
> Hi @CoreloneH, thanks for your interest in the repo. Definitely! I tried to tune hyperparameters in hinge loss but the FID score didn't converge and qualitative results were terrible. So, I tried `wgangp_eps` and in a few epochs, it converged very well compared to hinge. You can find the details in `train.py`.

Hi, thanks for your reply, I think the reason lies in that hinge loss fail to satisfy the Lipschitz continuity,
